cypress/e2e/App.e2e.ts/App.cy.ts

```diff
@@ -0,0 +1,57 @@
+
+describe('rendering app', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/')
+  })
+
+  it('loading', () => {
+    cy.request({
+      url: 'http://localhost:4000/movies?limit=9?',
+      followRedirect: false,
+    }).then((resp) => {
+      expect(resp.status).to.eq(200)
+    })
+
+  })
+  it('header component', () => {
+    cy.get('header').within(($header) => {
+      cy.get('label').should('have.text', 'netflixroulette')
+      cy.get('button').should('have.class', 'Button_add_movie__r2vPF')
```

Comment:
your e2e tests fails, you use css modules so classnames change often. You need to find better query. Also e2e tests are for covering user scenarios so it should follow a `user scenario` not verify if some classname exists or not. You can use unit tests for that. 
![image](https://user-images.githubusercontent.com/29926810/226293556-713d94c4-1a49-43fb-9f4c-2d63570536e7.png)